website/docs/d/continuous_compliance_policy.html.markdown

```diff
@@ -28,8 +28,8 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `cloud_account_id` - GCP account name in Dome9.
-* `external_account_id` - The account number.
-* `cloud_account_type` - creation date for project in Google.
-* `bundle_id` - Organizational unit id.
+* `target_internal_id` - GCP account name in Dome9.
+* `target_external_id` - The account number.
+* `target_type` - creation date for project in Google.
+* `ruleset_id` - Organizational unit id.
```

Comment:
this entire section is wrong I think. what does Google has to do with this documentation?

Reply:
As an example, it doesn't matter which vendor it is.